packages/docs/src/Header.tsx

```diff
@@ -15,7 +15,27 @@ import { Licenses } from './Licenses'
 const THEME_OPTIONS: ReadonlyArray<{
   readonly value: ThemeName
   readonly label: string
-}> = [{ value: ThemeName.DEFAULT_THEME, label: 'Default theme' }]
+}> = [
+  { value: ThemeName.BROWN, label: 'Brown' },
+  { value: ThemeName.GREY, label: 'Grey' },
+  { value: ThemeName.BLUE_GREY, label: 'Blue grey' },
+  { value: ThemeName.RED, label: 'Red' },
+  { value: ThemeName.PINK, label: 'Pink' },
+  { value: ThemeName.PURPLE, label: 'Purple' },
+  { value: ThemeName.DEEP_PURPLE, label: 'Deep purple (default)' },
+  { value: ThemeName.INDIGO, label: 'Indigo' },
+  { value: ThemeName.BLUE, label: 'Blue' },
+  { value: ThemeName.LIGHT_BLUE, label: 'Light blue' },
+  { value: ThemeName.CYAN, label: 'Cyan' },
+  { value: ThemeName.TEAL, label: 'Teal' },
+  { value: ThemeName.GREEN, label: 'Green' },
+  { value: ThemeName.LIGHT_GREEN, label: 'Light green' },
+  { value: ThemeName.LIME, label: 'Lime' },
+  { value: ThemeName.YELLOW, label: 'Yellow' },
+  { value: ThemeName.AMBER, label: 'Amber' },
+  { value: ThemeName.ORANGE, label: 'Orange' },
+  { value: ThemeName.DEEP_ORANGE, label: 'Deep orange' },
+]
```

Comment:
How would you feel about rewriting this to:

```typescript
const THEME_OPTIONS: ReadonlyArray<{
  readonly value: ThemeName
  readonly label: string
}> = [
  ...Object.entries(ThemeName).map(([, value]) => {
    if (value === ThemeName.DEEP_PURPLE) {
      return { value, label: `${value} (default)` }
    }

    return { value, label: value }
  }),
]
```

Would make the theme names "ugly" but correct.

Reply:
Actually Object.values() would suffice.